data/applications/mlpipelines.yaml

```diff
@@ -0,0 +1,17 @@
+metadata:
+  name: mlpipelines
+  annotations:
+    opendatahub.io/categories: 'ML Pipelines'
+spec:
+  displayName: ML Pipelines
+  provider: Kubeflow
+  description: |-
+    Pipeline tool specialized for Machine Learning workloads, leveraging Kubeflow and Openshift Pipelines
+  kfdefApplications: ['ml-pipelines']
+  route: ml-pipeline-ui
+  img: images/kubeflow.svg
+  category: Red Hat managed
+  support: red hat
+  docsLink: https://www.kubeflow.org/docs/components/pipelines/
+  quickStart: create-ml-pipeline
+  getStartedLink: https://www.kubeflow.org/docs/started/
```

Comment:
This should link directly to a getting started doc for for kfp-tekton.  We should create an issue to add getting started documentation to opendatahub.io

Reply:
The kfp-tekton 'Getting Started Doc' is just a section in the README in the kfp-tekton repo (https://github.com/kubeflow/kfp-tekton).  Given it doesn't have a true user documentation look and feel, is that what we should be referencing here?

FWIW this PR also contains our own simple getting started doc (`data/getting-started/mlpipelines.md`), but it appears that is used in the documentation Tiles, not the one containing the App Launcher.  Also, the `getStartedLink` referenced here points to an absolute/external URL, unlike the `quickStart` item that references internal docs. Is there a way to reference internal docs similarly here?